data/shaders/postprocess.frag

```diff
@@ -47,6 +47,19 @@ vec3 aces_fitted(vec3 color) {
     return clamp(color_tonemapped, 0.0, 1.0);
 }
 
+const float A = 0.15;
+const float B = 0.50;
+const float C = 0.10;
+const float D = 0.20;
+const float E = 0.02;
+const float F = 0.30;
+const float W = 11.2;
+
+vec3 Uncharted2(vec3 color) {
```

Comment:
huh

Reply:
Is this like Uncharted2's tone mapping?

Reply:
Yeah, I was playing around with ones that made the scene not too dark. ACES is pretty dark, will need exposure as well. This one with exposure factor 16 makes it quite bright. You can play around and see which you'd prefer.

Reply:
bet